sdks/python/apache_beam/transforms/core.py

```diff
@@ -698,7 +703,7 @@ def merge_accumulators(self, accumulators, *args, **kwargs):
 
     class ReiterableNonEmptyAccumulators(object):
       def __iter__(self):
-        return itertools.ifilter(filter_fn, accumulators)
+        return filter(filter_fn, accumulators)
```

Comment:
I think this might be a potential source for the performance loss --> I'll update this to use ifilter on PY2

Reply:
@charlesccychen and @tvalentyn: is there more detailed info on the benchmarks?